### PR TITLE
cpu_topology_test: fix wrong cpu topology

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 2034961

Signed-off-by: nanliu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@PaulYuuu Please help to review this, thanks.

(1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.cpu_topology_test.q35: PASS (227.79 s)

File: qemu/tests/cpu_topology_test.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def check(p_name, exp, check_cmd):
     for vcpu_threads in vcpu_threads_list:
         vcpu_sockets = min(max(host_cpu // (vcpu_cores * vcpu_threads), 1),
                            random.randint(1, 6))
-        vcpu_sockets = 2 if (os_type == 'Windows' and
+        vcpu_sockets = 2 if (os_type.lower() == 'windows' and
```
Comment (PaulYuuu) [inline qemu/tests/cpu_topology_test.py:49]: `os_type` should always be `windows`.
https://github.com/avocado-framework/avocado-vt/blob/c4c4529d5c4f3d095f42f28a3cb4f9f4856b97cb/virttest/shared/cfg/guest-os/Windows.cfg#L6

Comment (nanliu-r) [reply qemu/tests/cpu_topology_test.py:49]: > `os_type` should always be `windows`. https://github.com/avocado-framework/avocado-vt/blob/c4c4529d5c4f3d095f42f28a3cb4f9f4856b97cb/virttest/shared/cfg/guest-os/Windows.cfg#L6

@PaulYuuu  ok, thank you, has updated the commit message, please help to review this, thanks.

**Comment by nanliu-r** [comment]
(1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.cpu_topology_test.q35: PASS (229.62 s)

**Review by PaulYuuu — APPROVED** [review]
ACK

**Pull request merged**